tests/datenkompass/test_transform.py

```diff
@@ -36,12 +43,68 @@ def test_get_vocabulary() -> None:
     assert result == ["Infektionskrankheiten und -epidemiologie"]
 
 
+def test_get_datenbank() -> None:
+    item = mocked_merged_bibliographic_resource()[0]
+
+    assert get_datenbank(item) == (
+        "https://doi.org/10.1234_find_this_first, find_second_a, "
+        "find_second_b, https://www.find_third.to"
+    )
+
+
 def test_transform_activities() -> None:
     extracted_and_filtered_merged_activities = mocked_merged_activities()[
         :2
     ]  # item with no BMG filtered out
-    all_units = mocked_merged_organizational_units()
+    merged_organizational_units = {
+        unit.identifier: unit for unit in mocked_merged_organizational_units()
+    }
 
-    result = transform_activities(extracted_and_filtered_merged_activities, all_units)
+    result = transform_activities(
+        extracted_and_filtered_merged_activities, merged_organizational_units
+    )
 
     assert result == mocked_datenkompass_activity()
+
+
+@pytest.mark.usefixtures("mocked_backend_api_connector")
+def test_transform_bibliographic_resource() -> None:
+    extracted_and_filtered_merged_bibliographic_resource = (
+        mocked_merged_bibliographic_resource()
+    )
+    merged_organizational_units = {
+        unit.identifier: unit for unit in mocked_merged_organizational_units()
+    }
+    person_name_by_id = {
+        person.identifier: person.fullName for person in mocked_merged_person()
+    }
+
+    result = transform_bibliographic_resources(
+        extracted_and_filtered_merged_bibliographic_resource,
+        merged_organizational_units,
+        person_name_by_id,
+    )
+
+    assert result == [
```

Comment:
```suggestion
    assert result[0].model_dump() == [
```
We typically compare the dumped dictionaries. It's optional here though as we don't get into trouble with the Identity provider here

Reply:
I have a question here. Let's meet?